core-features/studio/prisma/schema.prisma

```diff
@@ -11,8 +11,4 @@ model User {
     id    Int    @id
     name  String
     email String
-
-    parentId Int
-    parent   User   @relation(fields: [parentId], references: [id])
-    children User[] @relation("UserToUser")
```

Comment:
Was just causing problems while deleting etc. I have a test for self-referential models in Studio, so all good